pytest_splunk_addon_ui_smartx/components/controls/single_select.py

```diff
@@ -130,16 +190,36 @@ def list_of_values(self):
         self.dropdown.click()
         first_element = None
         list_of_values = []
+
+        if self.allow_new_values:
+            if self.seachable:
+                self.elements.update({
+                    "input": Selector(select=self.container.select + ' [data-test="textbox"]')
+                })
+            popoverid = '#' + self.combobox.get_attribute("data-test-popover-id")
+            self.elements.update({
+                "values":Selector(select=popoverid + ' [data-test="option"]')
+            })
+        else:
+            if self.searchable:
+                popoverid = '#' + self.dropdown.get_attribute("data-test-popover-id")
+                self.elements.update({
+                        "input": Selector(select=popoverid + ' [data-test="textbox"]')
+                    })
+            popoverid = '#' + self.dropdown.get_attribute("data-test-popover-id")
```

Comment:
this statement is getting repeated 2 times in the else block.. Can we move it outside ?